config/config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "ListenPort" : ":8080",
+    "Log" : {
+        "FileName" : "Misha",
```

Comment:
rename FileName to logs.log

Reply:
Done.

Reply:
isn't done, you still didn't commit or push changes

Reply:
Done!